precompiled/serde_derive/src/lib.rs

```diff
@@ -15,8 +15,18 @@
 
 #![doc(html_root_url = "https://docs.rs/serde_derive/1.0.183")]
 
-#[cfg(not(all(target_arch = "x86_64", target_os = "linux", target_env = "gnu")))]
+#[cfg(all(
+    serde_derive_build = "source",
+    target_arch = "x86_64",
+    target_os = "linux",
+    target_env = "gnu"
+))]
 include!("lib_from_source.rs");
 
-#[cfg(all(target_arch = "x86_64", target_os = "linux", target_env = "gnu"))]
+#[cfg(all(
+    not(serde_derive_build = "source"),
```

Comment:
`cfg(all(not(…` → `cfg(not(all(…`

Reply:
@Kijewski this is not correct!

Reply:
Hey chill up. There seems to be some difficulty I'm debugging with environment variables.